### PR TITLE
Assimp integration

### PR DESCRIPTION
**Pull request by ydtqhd**
This PR adds:
- load player models from FBX files with Assimp
- single animation playback for Assimp player models
- store and use material color and texture of each mesh for Assimp player models
- in-memory image loading for `Texture` class

Loading textures from FBX file only supports embedded compressed images, meaning that model and texture images are contained in a single FBX file. It has only been tested on JPEG images, which should work for all available skins in assets we have so far.

Rename `AssimpScene.inl` to `Scene.inl` to render with Assimp loader.

### CODE REVIEW

**Review by mattlawr — DISMISSED** [review]
awesome, looks like it will work fine!

File: src/client/main_assimp.cpp
Comment (yesean) [inline src/client/main_assimp.cpp:1]: is this file still needed?

**Review by yesean — APPROVED** [review]
lgtm
